modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/WFSTestData.java

```diff
@@ -98,26 +101,26 @@
     }
 
     public static final TestDataType GEOS_ARCHSITES_11 = new TestDataType("GeoServer_2.0/1.1.0",
-            new QName("http://www.openplans.org/spearfish", "archsites"), "sf:archsites",
+            new QName("http://www.openplans.org/spearfish", "archsites"), "sf_archsites",
```

Comment:
This really does not look right ... I would expect "archsites" as that is the name of the type in DescribeFeatureType?

The "sf" is only the prefix used to look up the http://www.openplans.org/spearfish namespace - why is it being part of the name.